test/test_losses.py

```diff
@@ -0,0 +1,229 @@
+import pytest
+import torch
+import torch.nn.functional as F
+from common_utils import cpu_and_gpu
+from torchvision import ops
+
+
+def get_boxes(dtype, device):
+    box1 = torch.tensor([-1, -1, 1, 1], dtype=dtype, device=device)
```

Comment:
Not super happy with this choice of box. Since this is actually invalid input

Reply:
Agree with the concern.

Detectron2 used the same set of boxes. [see this](https://github.com/facebookresearch/detectron2/blob/42c44b622eb7b179576bdac218c84c107fff8eab/tests/layers/test_losses.py#L9). 

I think we should use valid input boxes.

That being said, should we also check if the input boxes have non-negative values? 
What do you think?

Reply:
We cannot assert here as it will lead to cuda call and cause trouble. 

I'm not sure if we can use `torch._assert_async` either.

Reply:
hmm, ig this situation is similar to  https://github.com/pytorch/vision/pull/5776#discussion_r844893521